suite-native/module-accounts/src/components/AccountDetailGraphHeader.tsx

```diff
@@ -41,13 +46,21 @@ const hasPriceIncreasedAtom = atom(get => {
     return percentageChange >= 0;
 });
 
-const CryptoBalance = ({ accountSymbol }: { accountSymbol: NetworkSymbol }) => {
+const CryptoBalance = ({ accountKey }: { accountKey: AccountKey }) => {
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+    const isTestnetAccount = useSelector((state: AccountsRootState) =>
+        selectIsTestnetAccount(state, accountKey),
+    );
     const selectedPoint = useAtomValue(selectedPointAtom);
 
+    if (!account) return null;
+
     return (
         <CryptoAmountFormatter
-            value={selectedPoint.cryptoBalance}
-            network={accountSymbol}
+            value={isTestnetAccount ? account.formattedBalance : selectedPoint.cryptoBalance}
```

Comment:
Please never use anything with `formatted` prefix or suffix. Purpose of formatters is that it's only place where you format values and they should always receive raw values.

Reply:
Ok, my bad. I copy pasted that from `TransactionListItem`. I removed `formattedBalance` from both places in 274ce39a2.